crafts.lua

```diff
@@ -28,7 +28,7 @@ minetest.register_craft({
 	}
 })
 
-if minetest.settings:get_bool("bridger_enable_trusses") then
+if minetest.settings:get_bool("bridger_enable_trusses") or true then

```

Comment:
The `or true` makes it so these settings cannot be disabled, because if they're `false`, `false or true` yields `true`.  So, do:

```lua
if minetest.settings:get_bool("bridger_enable_trusses") ~= false then
```

Reply:
You would think I would remember how lua works by now... Fixed :)